### ISSUE TITLE
multi: Switch from trezor.js to trezor/connect.

### ISSUE THREAD

**Issue by JoeGruffins** [decred/decrediton#2680]
The purpose of this issue is to track the changes needed to switch from trezor.js to connect.

Problem:

trezor.js is being deprecated.

https://github.com/trezor/trezor.js
```
warning OBSOLETE. DO NOT USE! Use https://github.com/trezor/connect instead 
```

As such, decrediton should switch to using connect.

Solution:

Work has been done on connect, and trezor-utxo-lib, a library used by connect, to support decred.
https://github.com/trezor/connect/pull/671
https://github.com/trezor/trezor-utxo-lib/pull/9

No changes are needed in the trezor firmware to switch. However, it looks like we may be waiting a while for the changes to connect and the utxo library to be reviewed/merged. So, we will not be able to use the official repositories for now. If that is a problem, these changes will just have to wait indefinitely. Hopefully it is not.

I believe all of the methods needed have been implemented as a proof of concept here:
https://github.com/matheusd/trezor-decred-poc/pull/5

It does not look like it is possible to connect with both trezor.js and trezor/connect to the same device at the same time. I would prefer to make all the changes in one pr for simplicity's sake. Albeit a big one.

**Comment by JoeGruffins** [comment]
I would like to work on this.

**Comment by JoeGruffins** [comment]
Solved in #2705

**Comment by xaur** [comment]
Sorry for a noob question but does it make Decrediton connect to `connect.trezor.io` (connect's [README](https://github.com/trezor/connect/blob/develop/README.md) mentions this host)?
